modulefiles/gfsutils_hera.lua

```diff
@@ -0,0 +1,17 @@
+help([[
```

Comment:
I don't think this is needed. Instead, the other modulefiles need to be renamed to include the `COMPILER` in their name similar to the `gfsutils_hera.intel.lua` name.